frontend/isoko/src/pages/Home.tsx

```diff
@@ -35,7 +35,6 @@ const TopContent = styled.div`
    min-height: 500px;
 `;
 
-// TODO: Combine these three into a single component taking in props
```

Comment:
Didn't know where else to put this comment but I think you should add the hamburger component to the home page. things like business dashboard, profile, signup should be available from home page.